sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<p>Looks like nested numbering is not supported. (In both notebooks)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Seems to be working when rendered by github https://github.com/maaquib/amazon-sagemaker-examples/blob/triton/sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb

Not sure why it is not working in ReviewNB

Reply:
Thanks for the link, you can disregard this comment given that it seems to be a ReviewNB issue.